SleepNumberController_App.groovy

```diff
@@ -58,7 +58,7 @@ preferences {
   page name: "createBedPage"
   page name: "diagnosticsPage"
 }
-
+/*
```

Comment:
this is, iirc, required to handle the pause/resume button at the top of the app.

Reply:
I don't see any references.  It looks like a ST thing to me.

Reply:
Definitely a thing ... I just commented it out locally and tried clicking the pause button and HE emits this error:

2020-12-17 03:36:04.092 pm errorgroovy.lang.MissingMethodException: No signature of method: user_app_rvrolyk_Sleep_Number_Controller_898.appButtonHandler() is applicable for argument types: (java.lang.String) values: [pause] (appButtonHandler)